libs/application/templates/parental-leave/src/lib/dataSchema.ts

```diff
@@ -51,13 +50,11 @@ export const dataSchema = z.object({
   periods: z.array(Period).nonempty(),
   employer: z.object({
     name: z.string().nonempty(),
-    nationalRegistryId: z
-      .string()
-      .nonempty()
-      .refine(
-        (n) => kennitala.isValid(n) && kennitala.isCompany(n),
-        'Kennitala þarf að vera gild',
-      ),
+    nationalRegistryId: z.string().nonempty(),
+    // .refine(
```

Comment:
Should we add a note for the commented out code?

Reply:
this will be removed in Bærings work